conda_forge_tick/make_graph.py

```diff
@@ -352,6 +352,25 @@ def update_nodes_with_bot_rerun(gx):
                     )
 
 
+def update_nodes_with_new_versions(gx):
+    """Updates every node with it's new version (when available)"""
+    # check if the versions folder is available
+    if os.path.isdir("./versions"):
+        # get all the available node.json files
+        pass
+    else:
+        return
+    list_files = os.listdir("./versions/")
+    for file in list_files:
+        node = str(file).replace(".json", "")
+        with open(f"./versions/{file}") as json_file:
+            version_data = json.load(json_file)
+        with gx.nodes[f"{node}"]["payload"] as attrs:
+            attrs["new_version"] = version_data.get("new_version")
```

Comment:
`attrs.update(**version_data))`

Reply:
If that doesn't work please add an update method to `LazyJSON`

Reply:
I will try that, but I will have to think a way to test that though

Reply:
Should I add the bad feedstocks too for `update_versions` ? to solve the problems with the outdated `.json` files ?

Reply:
Or let it run for now and see that after ?

Reply:
I was looking into the `LazyJSON` config at `.utils` and there isn't any `.update` method there, so would this be enough ?
```python
    def __update__(self, data: dict) -> None:
        for key in data.items():
            self.__dict__[f'{key}'] = data[f'{key}']
```